scripts/bash/backendStartDev.sh

```diff
@@ -39,7 +39,7 @@ done
 
 # Start server command for TS
 if [[ ${type_script} == true ]]; then
-    start_server="nodemon -w src --exec \"babel-node --extensions \".ts\" src/index.ts --inspect=${inspect_port} --config-file '../../.babelrc-ts.js'\""
+    start_server="nodemon -w src -e ts,json --exec \"babel-node --extensions \".ts\" src/index.ts --inspect=${inspect_port} --config-file '../../.babelrc-ts.js'\""
```

Comment:
For fixing an issue that start-dev psss-server does not hot reload when skipping internal dependencies

Reply:
Nice one!